src/grammar-parser/grammar-parser.jison

```diff
@@ -136,6 +140,12 @@ expression
   | FUNCTION '(' ')' {
       $$ = yy.callFunction($1);
     }
+  | NUMBER 'e' '-' NUMBER {
```

Comment:
I had to do a special case here for negative exponents due to the overloading of unary minus and the minus sign